source/develop/release-management/features/sla/external-scheduling-proxy.md

```diff
@@ -51,31 +50,32 @@ This feature will also allow new feaures to build on it.
 
 Engine and external scheduler API:
 
-*   Discover(void): returns a XML containing all available policy units and configurations (configuration is optional).
+*   `Discover(void)`: returns a XML containing all available policy units and configurations (configuration is optional).
 
 Discover will iterate all plugins and config files and extract the data.
 
 Sample of data returned by the discover function:
 
-`     `<PolicyUnits>
-`       `<Filters>
-`         `<Filter>
-`           `<name>`Memory`</name>
-`           `<Properties>
-`             `<name>`Heat`</name>
-`             `<CustomProperties>`server_ip=\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b;threshold={0-99}`</CustomProperties>
-`             `<default_values>`127.0.0.1;70`</default_values>
-`           `</Properties>
-`         `</Filter>
-`       `</Filters>
-`       `<CostFunctions>
-`         `<CostFunction name="Memory"/>
-`       `</CostFunctions>
-`       `<Balances>
-`         `<Balance name="Memory"/>
-`       `<Balances>
-`     `</PolicyUnits>
-         
+{% highlight xml %}
```

Comment:
This does not look right in the GitHub rendered form, not sure what the markdown processor you use will do.

Reply:
Attached screenshot